components/builder-jobsrv/habitat/default.toml

```diff
@@ -7,6 +7,8 @@ worker_heartbeat_listen = "0.0.0.0"
 worker_heartbeat_port = 5567
 publisher_listen = "0.0.0.0"
 publisher_port = 5568
+log_ingestion_listen = "0.0.0.0"
+log_ingestion_port = 5569
```

Comment:
Ah, I didn't realize that. Probably best to remove that and reset port numbers in a separate PR, then 😄